packages/bun-types/ambient.d.ts

```diff
@@ -23,3 +23,9 @@ declare module "*.html" {
   var contents: any;
   export = contents;
 }
+
+declare module "*.svg" {
+  // Bun 1.2.3 added support for frontend dev server
+  var contents: `${string}.svg`;
```

Comment:
nice way to say "this is a file path"

is there a way we could let users override it though? like if they use the text loader to import SVGs or something

Reply:
I think for this one it should create a .d.ts file with it for bun init, as otherwise you will have to add every common extension

Reply:
@RiskyMH are you saying similar to like `next-env.d.ts`?

Reply:
Yeah it should show people how to add other extensions

but tbh still could be fine to add a bigger default list of extensions (I don't think ts validates path so could give too much false hope though)

Reply:
Yeah it doesn't validate it exists unfortunately. Our declaration is the contract that it exists.
I think it's ok in this case, and maybe we can write some doc about "If you want to support .myfile you can define bun-env.d.ts or similar"

Reply:
A doc for this would be helpful -- with `bun-types@1.2.5`, I'm running into the TypeScript issue:

```
Type '`${string}.svg`' is not assignable to type 'ComponentType<PropsWithChildren<SVGProps<SVGElement>>>'.
```

and I'm still not sure how to fix it.

Reply:
@michaelhays @chimericdream I've opened #18527 to address this